src/main/java/com/google/api/generator/engine/ast/ClassDefinition.java

```diff
@@ -78,8 +81,12 @@ public static Builder builder() {
         .setNestedClasses(Collections.emptyList());
   }
 
+  public abstract Builder toBuilder();
+
   @AutoValue.Builder
   public abstract static class Builder {
+    public abstract Builder setFileHeader(List<CommentStatement> fileHeader);
```

Comment:
Does it make sense to error-out in `build()` if this is a nested class?

Reply:
yes, added one condition check in `build` to confirm nested classes do not have file headers.